### ISSUE TITLE
Fingerprinting resistance mode

### ISSUE THREAD

**Issue by snyderp** [privacycg/proposals#8]
Most browsers have a variety of similar-but-different fingerprinting protections, which make things unpredictable for site authors and cause web-compat problems on all browsers.

This deliverable would be for PrivacyCG to (i) document, (ii) standardize and (iii) extend these fingerprinting protections.

This deliverable would aim (i) to give web authors had a common set of expectations to target, (ii) make it easier for browser users to understand protections in platform, and (iii) reduce web compat problems fingerprinting protections cause, though a "strength in numbers" approach.

A "fingerprinting resistance mode" wouldn't need to be an explicit "mode" either.  Some vendors might wish to have "fingerprinting resistance" enabled all the time, others might alias it with "private browsing", etc.

**Comment by snyderp** [comment]
@hober yes, thanks! (and apologies, won't make the error again)

**Comment by hober** [comment]
Marked `agenda+` per discussion in #privacycg slack.

**Comment by othermaciej** [comment]
> The only think I'd add yours are threshold / budget style approaches (Privacy Budget and PrivacyBadger for examples). I don't think that is a promising approach, but worth noting maybe.

Agreed, both that it's worth noting and that it may not be very promising.

> 2. Make formal objections to new versions of the relevant specs until the FP surface has been dealt with in the spec

I think we should start with milder forms of feedback on the relevant specs, such as filing issues. If there is cross-browser agreement on a change, it shouldn't be that hard to get it in. Also WHATWG doesn't have such a thing as formal objections, and many changes will likely need to be in specs developed there. (There is a way to appeal to the Steering Group but it's never been used and I hope it stays that way).

I'm wondering if this has overlap with PING work to do privacy reviews of existing specs.

**Comment by snyderp** [comment]
> I think we should start with milder forms of feedback on the relevant specs, such as filing issues.

Most definitely :)  I was jumping a head one step ahead in my head based on past experience, but you're def right, issues first!

**Comment by AramZS** [comment]
While I agree consistent fingerprinting resistance makes sense I think there is a need to be aware of the current use-cases in the marketplace that may need some degree of support in the future. The question here is how (or if) to retain some degree of anti-fraud protection which is one of the (theoretically) privacy-respecting ways that fingerprinting is being used. Even if the position is "we don't want to support that use case" we should still be aware of it and be able to articulate why not and if there is an interest in presenting an alternative.

**Comment by hober** [comment]
Hi @pes10k! Any appetite to come back to this? I think the anti-fingerprinting browser feature landscape has changed at least some in the last couple of years.

**Comment by pes10k** [comment]
I think there is probably less need for this, since PING and TAG (at least it seems to me) have been more aggressive (in a good and welcome way) in preventing passive fingerprinting surface for winding up in new proposals.

If vendors would be interested in a "fingerprinting resistance" mode to standardize privacy protections to deal with legacy features, though, that could be useful, especially all browsers have some "tracking resistance" mode now, though with different names and framing.

Are other vendors interested?  If so, i think its worth continuing this; if not, i think fine to archive and close.

**Comment by annevk** [comment]
Mozilla would certainly be interested in exploring this space further. Ideally things that can be shipped to all users, but engine-wise we have some support for opt-in fingerprinting resistance as well.

cc @tomrittervg

**Comment by spanicker** [comment]
Chrome doesn’t have a resist fingerprinting mode right now, but we’re happy to join the conversation.
